src/ucp/proto/proto_init.c

```diff
@@ -279,10 +272,12 @@ ucp_proto_init_add_buffer_copy_time(ucp_worker_h worker, const char *title,
                                     size_t range_start, size_t range_end,
                                     int local, ucp_proto_perf_t *perf)
 {
-    ucp_proto_perf_factors_t perf_factors = UCP_PROTO_PERF_FACTORS_INITIALIZER;
-    ucp_context_h context                 = worker->context;
-    ucp_proto_perf_factor_id_t factor_id  = ucp_proto_buffer_copy_factor_id(
-            local_mem_type, remote_mem_type, memtype_op, local);
+    ucp_context_h context                            = worker->context;
+    ucp_proto_perf_factors_t perf_factors            =
+            UCP_PROTO_PERF_FACTORS_INITIALIZER;
+    ucp_proto_perf_factor_id_t buffer_copy_factor_id =
```

Comment:
minor: initialize it after the declaration to reduce indentation

Reply:
Done